test/middlewares/error-middleware.test.js

```diff
@@ -0,0 +1,135 @@
+'use strict';
+
+/**
+ * Module dependencies.
+ */
+
+const GrpcError = require('grpc-error');
+const Mali = require('mali');
+const caller = require('grpc-caller');
+const errorMiddleware = require('src/middlewares/error-middleware');
+const host = 'localhost:50051';
```

Comment:
Nitpick: move `host` and `proto` to a `Constants` block? Since these are not really dependencies.